lib/elixir/lib/application.ex

```diff
@@ -145,6 +145,19 @@ defmodule Application do
   """
   @callback stop(state) :: term
 
+  @doc """
+  Called after an application starts.
+
+  This function is called after an application starts and before any included
+  applications are started.  It will be called once for every start phase defined
```

Comment:
Double space after the period. Not sure if intentional, but we tend not to have those :)